concepts/lifecycle.py

```diff
@@ -325,7 +315,43 @@ def get_matching(self):
             A list of dicts with raw data from Conceptpower.
         """
         try:
-            data = self.conceptpower.search(self.instance.uri)
+            data = self.get_concept(self.instance.uri)
         except Exception as E:
             raise ConceptUpstreamException("Whoops: %s" % str(E))
-        return list(map(self._reform, data))
+        return list(data)
+
+    def get_concept(self, uri):
+        try:
+            url = f"{settings.CONCEPTPOWER_ENDPOINT}Concept?id={uri}"
+            headers = {
+                'Accept': 'application/json',
+            }
+            response = requests.get(url, headers=headers)
+
+            if response.status_code == 200:
+                    data = response.json()
+                    concept_entry = data.get('conceptEntries', [{}])[0]
+            else:
+                raise ValueError(f"Error fetching concept data: {response.status_code}")
```

Comment:
same, there should be a conceptpower class that does all the communication with conceptpower (isn't there one already?)